ConnectFour/Part3_FrontEndProject.css

```diff
@@ -0,0 +1,8 @@
+.board button {
+  width: 100px;
+  height: 100px;
+  background-color: gray;
+  border-radius: 50%;
+  border: 4px solid black; /*this gets rid of bootstrap shadow*/
+  margin: 1px;
+}
```

Comment:
Use CSS beautifier.

Reply:
@chicken-biryani please review the changes.